sympy/solvers/solveset.py

```diff
@@ -1806,13 +1807,73 @@ def solvify(f, symbol, domain):
 ###############################################################################
 
 
+def linear_coeffs(eq, *syms):
```

Comment:
If it's not in the init file it won't be imported by `from sympy import *`; it would be imported with `from sympy.solvers.solveset import *`, however.